ArduCopter/AP_Arming.cpp

```diff
@@ -44,7 +44,9 @@ bool AP_Arming_Copter::run_pre_arm_checks(bool display_failure)
     }
 
     // always check motors
-    if (!motor_checks(display_failure)) {
+    char failure_msg[50] {};
+    if (!copter.motors->arming_checks(ARRAY_SIZE(failure_msg), failure_msg)) {
+        check_failed(display_failure, "Motors: %s", failure_msg);
```

Comment:
I only had the message notification sent in two parts.
This requires that all messages be put into 50 bytes.
The messages are sent in STATUSTEXT messages.
https://mavlink.io/en/messages/common.html#STATUSTEXT

Reply:
I agree that we should try and keep messages to display on a single line if possible.  We didn't consider this a blocker but maybe worth a follow-up PR but we will leave this to @IamPete1 to decide.